src/api-controller.js

```diff
@@ -0,0 +1,4 @@
+const getMovies = () => fetch('https://api.tvmaze.com/shows')
```

Comment:
I think it will be nice to use async functions and await as advised by Microverse in the previous similar task

Reply:
I used it in index.js file